airflow/models.py

```diff
@@ -3671,12 +3671,21 @@ def topological_sort(self):
 
     @provide_session
     def set_dag_runs_state(
-            self, state=State.RUNNING, session=None):
+            self,
+            state=State.RUNNING,
+            session=None,
+            only_include_backfill_dagruns=False):
```

Comment:
nit: `only_backfill_dagruns` would be clear enough

Reply:
updated